charts/ontology/templates/deployment.yaml

```
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}-deployment
+  labels:
+{{ include "ontology.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+  template:
+    metadata:
+      name: {{ .Chart.Name }}
+      labels:
```

```suggestion
      labels:
{{ include "ontology.labels" . | indent 8 }}
```
It's a good practice to have the chart template labels on the pods as well so that they can be used for easier selection/searching